### PR TITLE
Add a small section about Google Cloud Key Management and signing.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

**Comment by kacf** [comment]
@Michaelhobo: FYI

**Review by lluiscampos — APPROVED** [review]

File: 06.Artifact-creation/07.Sign-and-verify/docs.md [hunk 1]
```diff
@@ -120,3 +120,7 @@ For OS specific instructions on how to install and enable verification keys, vis
 - [Yocto Project - Building for production - Artifact signing and verification keys](../../05.System-updates-Yocto-Project/06.Build-for-production/docs.md#artifact-signing-and-verification-keys).
 
 - [Debian family - Customize Mender](../../04.System-updates-Debian-family/03.Customize-Mender/docs.md)
+
+## Cloud Key Management
+
+It is possible to sign Artifacts using keys in Cloud Key Management. Currently the mender-artifact tool supports Google Cloud Key Management. For more information see the [Google Cloud Key Management web pages](https://cloud.google.com/security-key-management?target=_blank), and the help screen of mender-artifact, available by running the command `mender-artifact sign --help`.
```
Comment (TheMeaningfulEngineer) [inline 06.Artifact-creation/07.Sign-and-verify/docs.md:126]: I like the clarification use in the [commit](https://github.com/mendersoftware/mender-artifact/pull/371/commits/1ad629620ee6b960d0726d593605d02006705cbd). Here is a suggestion:

 ```suggestion
It is possible to sign Artifacts using keys in Cloud Key Management, allowing developers to sign Mender Artifacts without ever accessing the private signing key. Currently the `mender-artifact` tool supports [Google Cloud Key Management](https://cloud.google.com/security-key-management?target=_blank), for more info check the help for the `gcp-kms-key`, available by running the command `mender-artifact sign --help`.
```

**Review by TheMeaningfulEngineer — COMMENTED** [review]
Looks good.
Added a suggestion to which would help me clarify what this was about. :)

File: 06.Artifact-creation/07.Sign-and-verify/docs.md (hunk 1 above)
Comment (kacf) [reply 06.Artifact-creation/07.Sign-and-verify/docs.md:126]: Good idea, I will add that!

**Review by estenberg — APPROVED** [review]

**Pull request merged**